### PR TITLE
Convert monitored trip alert timestamps to seconds

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

This PR adds a timestamp conversion to seconds for monitored trip alerts so that their effective date
displays with the correct year in the monitored trip editor.

For the first PR of the year, some snapshots needed to be updated.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by miles-grant-ibigroup — APPROVED** [review]
How was this not previously caught ???

**Comment by binh-dam-ibigroup** [comment]
> How was this not previously caught ???

See how much time we have been working on alerts lately!

File: lib/components/user/monitored-trip/trip-status-rendering-strategies/base-renderer.js [hunk 1]
```diff
@@ -50,8 +62,11 @@ export default function baseRenderer(monitoredTrip) {
     )
   }
 
-  // set some alert data if the matching itinerary exists
-  data.alerts = data.matchingItinerary && data.matchingItinerary.alerts
+  // Set some alert data if the matching itinerary exists.
+  // (Convert alert timestamps in milliseconds to seconds for display using alerts body.)
+  data.alerts =
+    data.matchingItinerary &&
+    data.matchingItinerary.alerts.map(withAlertTimestampsInSeconds)
```
Comment (daniel-heppner-ibigroup) [inline lib/components/user/monitored-trip/trip-status-rendering-strategies/base-renderer.js:69]: could you write this using null coalescing operator?

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Wow, good catch!

File: lib/components/user/monitored-trip/trip-status-rendering-strategies/base-renderer.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/components/user/monitored-trip/trip-status-rendering-strategies/base-renderer.js:69]: Did you mean optional chaining? As written, `data.alerts` can be `null` and the existing code supports that.

**Pull request merged**